modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,146 @@
+
+[,bash]
+----
+rpk acl create --operation write --topic private-software-versions --allow-role User
+----
+
+== Managing users and roles
+
+Redpanda allows you to manage your RBAC configurations through the use of RPK. The Redpanda admin API provides another means of interacting with RBAC. Examples of using rpk for each RBAC operation are given here.
+
+=== Create a role
+
+Creating a new role is a two-step process. First you define the role, giving it a unique and descriptive name. Second, you assign one or more ACLs to allow or deny access for the new role. This defines the permissions that are inherited by all users assigned to the role. It is possible to have an empty role with no ACLs assigned.
+
+Administrators use rpk to create a new role by issuing the command:
+
+[,bash]
+----
+rpk security role create [ROLE_NAME]
+----
+
+After the role is created, administrators create new ACLs and assign this role either allow or deny permissions. For example:
+
+[,bash]
+----
+rpk acl create ... --allow-role [ROLE_NAME]
+----
+
+=== Delete a role
+
+When a role is deleted, Redpanda carries out the following actions automatically:
+
+- All role ACLs are deleted.
+- All users assignments to the role are removed.
+
+Using rpk, administrators delete a role by issuing the command:
+
+[,bash]
+----
+rpk security role delete [ROLE_NAME]
+----
+
+=== Assign a role
+
+Administrators may assign a role to any principal created in their Redpanda instance. Principals are referred to using the format: `Type:Name`. Redpanda currently supports only the `User` type. If you omit the type, Redpanda assumes the `User` type by default.
+
+Using rpk, administrators assign a role to a principal by issuing the command:
+
+[,bash]
+----
+rpk security role assign [ROLE_NAME] [PRINCIPAL]
+----
+
+=== Unassign a role
+
+Administrators may remove a role assignment from a principal without deleting the role. Principals are referred to using the format: `Type:Name`. Redpanda currently supports only the `User` type. If you omit the type, Redpanda assumes the `User` type by default.
+
+Using rpk, administrators remove a role assignment from a principal by issuing the command:
+
+[,bash]
+----
+rpk security role unassign [ROLE_NAME] [PRINCIPAL]
+----
+
+=== Edit a Role
+
+Using rpk, administrators can modify an existing role by adding additional ACLs to it using the command:
+
+[,bash]
+----
+rpk acl create ... --allow-role [ROLE_NAME]
+----
+
+See xref:security/authorization/acl.adoc[Access Control Lists] for more information on defining ACLs.
```

Comment:
```suggestion
See xref:security/authorization/acl.adoc[access control lists] for more information on defining ACLs.
```
Looks like we have standardized usage based on the style guide? https://github.com/redpanda-data/docs-site/blob/main/meta-docs/STYLE-GUIDE.adoc#word-list